supplementary_style_guide/style_guidelines/structure.adoc

```diff
@@ -80,6 +80,11 @@ Not a complete sentence: This prerequisite is acceptable if all the other prereq
 
 To help readers find the information that they need or to confirm that they are in the right place, every module and assembly must include a _short description_, also called an _abstract_.
 
+[IMPORTANT]
+====
+Do not start a module with an admonition, which was common practice for a Technology Preview feature. The information contained in an admonition might not convey a summary of a feature. The styling of an admonition is designed to draw the reader's attention, so placing a short description before an admonition would not distract our readership.
```

Comment:
This seems a little long, do you think we need to go into as much explanation here? Making some suggestions to be more concise and straightforward. Feel free to adjust or add back anything if you think it's important.

```suggestion
Do not start a module with an admonition, even when adding the Technology Preview admonition. Always include a short description before including an admonition.
```

Reply:
Thanks, @bergerhoffer . Updated.

Reply:
Thanks, @bergerhoffer . I had do reinstall an OS last week, so I'm unable to push content to this PR. I created https://github.com/redhat-documentation/supplementary-style-guide/pull/500 instead.